examples/C++/DDS/ContentFilteredTopicExample/README.md

```diff
@@ -0,0 +1,22 @@
+# Content Filtered Topic Example
+
+This example extends the HelloWorld example to show how to use Content Filtered Topics.
+The example includes the use of two different Filter Factories.
+On the one hand the default SQL filter and on the other hand a custom filter defined in the example.
+
+## Execution instructions
+
+To launch this example open three different consoles:
+
+In the first one launch the Subscriber using the default SQL filter: ./DDSContentFilteredTopicExample default_subscriber
+In the second one launch the Subscriber using the custom filter: ./DDSContentFilteredTopicExample custom_subscriber
+Finally, in the third terminal launch the Publisher: ./DDSContentFilteredTopicExample publisher
```

Comment:
Suggestion:
* In the first one, launch the Subscriber using the default SQL filter: 
   ```bash
   ./DDSContentFilteredTopicExample default_subscriber
   ```
* In the second one, launch the Subscriber using the custom filter:
   ```bash
   ./DDSContentFilteredTopicExample custom_subscriber
   ```
* Finally, in the third terminal, launch the Publisher:
   ```bash
   ./DDSContentFilteredTopicExample publisher
   ```